javascript-data-model/javascript-data-model/order-history.js

```diff
@@ -0,0 +1,38 @@
+var orderHistory = [
+  {
+    items: 'JavaScript for impatient progammers',
```

Comment:
I would recommend setting `items` to contain an array of objects, then you could store one object for each item in the order, and the data related to that item